### ISSUE TITLE
🐛 VSCode does not visually display CSS linting by default

### ISSUE THREAD

**Issue by leotroche** [biomejs/biome-vscode#355]
### VS Code version

1.93.1

### Extension version

2.3.0

### Biome version

1.9.1

### Operating system

- [X] Windows
- [ ] macOS
- [ ] Linux

### Description

Since biome version 1.9.0, the CSS linter and formatter are enabled by default in biome.
https://biomejs.dev/blog/biome-v1-9/#stable-css-formatter-and-linter

However, the VSCode extension does not detect that the css linter is enabled by default unless it is specified in the `biome.json` file:

```json
"css": {
  "linter": {
    "enabled": true
  }
}
```

Before enabling css linter manually in `biome.json`:

![{3185DEFE-4D90-44E5-B6F5-4E43E5968898}](https://github.com/user-attachments/assets/1b9194fc-d91e-4c49-98b8-3b473150be1c)

After enabling css linter manually in `biome.json`:

![{8CF6E1BB-EE34-4FC2-BFA2-06985AC86F5F}](https://github.com/user-attachments/assets/7df3afbf-7d63-41c2-b424-dd2fbbfaf411)

Additional information: In the playground it is displayed correctly.

[Playground](https://biomejs.dev/playground/?files.index.css=aAAyACAAewAKAAkAZgBvAG4AdAAtAGYAYQBtAGkAbAB5ADoAIABzAGEAbgBzAC0AcwBlAHIAaQBmACwAIABtAG8AbgBvAHMAcABhAGMAZQAsACAAbQBvAG4AbwBzAHAAYQBjAGUAOwAKAH0ACgAKAGQAaQB2ACAAewAKAAkAdAByAGEAbgBzAGYAbwByAG0AOgAgAHUAbgBrAG4AbwB3AG4AKAAxACkAOwAKAH0ACgA%3D)

### Steps to reproduce

1. Create an `biome.json` file with `biome init`

2. Create an `index.css` file with this code:

```css
h2 {
  font-family: sans-serif, monospace, monospace;
}

div {
  transform: unknown(1);
}
```

The absence of the linter confirms the bug.

If the following configuration is added to `biome.json`, the linter displays correctly:

```json
"css": {
  "linter": {
    "enabled": true
  }
}
```

### Expected behavior

VSCode should display errors on screen by default, without the need to manually enable css linter in `biome.json`.

### Does this issue occur when using the CLI directly?

No

### Link to a minimal reproduction

https://github.com/leotroche/biome-repro-1726486531097

**Comment by nhedger** [comment]
I believe this has been fixed in the latest patch release of Biome (1.9.3).

I was able to reproduce the issue with 1.9.1, and updating to 1.9.3 fixed the issue.
